managed/services/user/user.go

```diff
@@ -125,3 +125,23 @@ func (s *Service) UpdateUser(ctx context.Context, req *userpb.UserUpdateRequest)
 	}
 	return resp, nil
 }
+
+// ListUsers lists all users and their details.
+func (s *Service) ListUsers(_ context.Context, _ *userpb.ListUsersRequest) (*userpb.ListUsersResponse, error) {
```

Comment:
seems like we should call this endpoint somehow else since we don't return any other details except role ID

Reply:
I imagine this endpoint will be used in the future and we'll add more details to the response.
I did not want to create a separate endpoint just for the IDs and then create a new endpoint (or migrate) when we need more details.

Happy to move it to its own endpoint though if it's preferred.

Reply:
I mean that we already have more details we can gather from grafana, but don't include them, even the Username.

Reply:
Do you have in mind we shall retrieve the relevant details by default and provide them in the response?

Reply:
let's add it to TODO and discuss with @matejkubinec whether will it be easier for him as a consumer of this API endpoint.

Reply:
Yeah, currently the only use for it for the roles. We could evaulate this decision in the future, but I think we want to keep as much stuff separate from grafana as possible.

Reply:
ok, let's keep it as is then.